### ISSUE TITLE
Final Project Submission

### ISSUE THREAD

**Issue by lorifranke** [yoseple/cs460final#1]
![FeedbackBanner](https://github.com/yoseple/cs460final/assets/38534852/4c4a6482-1d77-4d80-8ecd-47731e2e32bd)

Hello @yoseple ,

Thanks for submitting the final project! Here is some feedback on your submission:

Here are the things you did great:

- You have done a great job in crediting sources and citing references/literature. This is an essential practice in academic and professional work.
- Providing instructions for using Flask demonstrates attention to detail and helps in making your project more accessible.

Here are some suggestions and possible improvements:

**Code and features**

The code is well structured. The folder with the nifti files could maybe be named 'data' and the 'template' folder could also maybe have another name since it contains the html files. I would also recommend not committing .DS_Store to github.

**Docs**

Adding a screenshot to the README would be beneficial! It could provide a quick visual overview of your project / making it more engaging for viewers.
Although the lack of a GitHub Pages deployment is understandable due to the Flask API, consider alternative ways to showcase your project online. For instance, a brief video demonstration, a GIF or a series of screenshots would be a suggestion :-)

**Overall Thoughts**

In summary, your project is very solid and can help researchers in the medical field! Thanks for adding a good documentation practices and clear instructions for using Flask! Including a visual element in the README would further elevate your project's appeal. Despite the absence of a GitHub Page, your project still stands out for its functionality and well-documented code. Keep up the good work Joseph, and thank you for your thoughtful and well-executed submission!
